apps/website/get-started/design.md

```diff
@@ -21,7 +21,7 @@ The default font intended for use with Clarity is [Clarity City](https://github.
 Clarity Angular uses Metropolis as the base font, which you can download below. Clarity City is an improved version of Metropolis, but has not been updated in Clarity Angular due to the potential for breaking changes.
 
 <div cds-layout="m-t:md">
-<a class="btn btn-primary asset-download-btn" target="_blank" href="https://github.com/vmware/clarity/files/5425574/Metropolis.zip"><cds-icon shape="download"></cds-icon> Clarity City (1.0.0)</a>
+<a class="btn btn-primary asset-download-btn" target="_blank" href="https://github.com/vmware/clarity/files/5425574/Metropolis.zip"><cds-icon shape="download"></cds-icon> Metropolis (1.001)</a>
```

Comment:
Thanks for catching this, but it should probably read `Metropolis M8`, where M8 is the version we originally used and subsequent updates cause layout issues.

Reply:
Thanks, I wasn't sure which version to use so I referred to the file metadata, updated!